index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Preview Card</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div class="all-content">
```

Comment:
🍒 As this is the only and main div in the page kindly use main tag for better HTML semantics